web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -222,6 +222,12 @@ fixmystreet.update_list_item_buttons = function($list) {
   $list.children(':last-child').find('[name="shortlist-down"]').prop('disabled', true);
 };
 
+fixmystreet.run = function(fn) {
+    if (fn) {
+        fn.call(this);
+    }
+};
+
```

Comment:
I saw this being called below and didn't understand what it was doing. I think I know now (it only calls a function if it exists, so it won't error for non-staff users), but it could do with a comment for clarification 👍